SIT.Manager.Avalonia/App.axaml

```diff
@@ -23,7 +24,7 @@
 	</Application.Resources>
 	
     <Application.Styles>
-		<sty:FluentAvaloniaTheme />
+		<sty:FluentAvaloniaTheme CustomAccentColor="#323232" />
```

Comment:
Do we really want to change this..... Makes some things really hard to see..... I'm not against the color being changed just not convinced a dark grey is the right choice.

With this change
![image](https://github.com/stayintarkov/SIT.Manager.Avalonia/assets/112902041/7f2a900b-446c-47e2-9755-d64729d0abd7)

Without:
![image](https://github.com/stayintarkov/SIT.Manager.Avalonia/assets/112902041/ee78a410-0f85-4be7-b562-65398006c9e9)



And probably a more obvious example of it being hard to see:
With:
![image](https://github.com/stayintarkov/SIT.Manager.Avalonia/assets/112902041/a1a97d35-8941-481c-8d55-27c70fccc708)

Without:
![image](https://github.com/stayintarkov/SIT.Manager.Avalonia/assets/112902041/d53fe2e9-f134-481c-b82e-39142fd3d60d)